src/query/functions/linear/datetime.go

```diff
@@ -25,27 +25,34 @@ import (
 	"math"
 	"time"
 
-	"github.com/m3db/m3/src/query/executor/transform"
+	"github.com/m3db/m3/src/query/block"
+	"github.com/m3db/m3/src/query/functions/lazy"
+
```

Comment:
nit: Join the imports?

Reply:
Good call; weird that the linter didn't break the build on this though